src/edits.ts

```diff
@@ -74,6 +75,50 @@ export async function runUpload(
     }
 }
 
+/**
+ * Check if the error is the specific Google Play API error requiring
+ * changesNotSentForReview=true.
+ * 
+ * The error message from Google Play API is:
+ * "Changes cannot be sent for review automatically. Please set the query 
+ *  parameter changesNotSentForReview to true."
+ * 
+ * @param error - The error object from the API call (typically a GaxiosError)
+ * @returns true if this is the specific error that can be handled by retry
+ * 
+ * @see https://developers.google.com/android-publisher/api-ref/rest/v3/edits/commit
+ */
+function isChangesNotSentForReviewRequired(error: unknown): boolean {
+    const gaxiosError = error as GaxiosErrorLike;
```

Comment:
Is `error` not a `GaxiosErrorLike`?

```suggestion
function isChangesNotSentForReviewRequired(error: GaxiosErrorLike): boolean {
```